nixos/modules/services/networking/hostapd.nix

```diff
@@ -906,7 +904,7 @@ in {
                   bssCfg = bssSubmod.config;
                   pairwiseCiphers =
                     concatStringsSep " " (unique (bssCfg.authentication.pairwiseCiphers
-                      ++ optionals bssCfg.authentication.enableRecommendedPairwiseCiphers ["CCMP" "CCMP-256" "GCMP" "GCMP-256"]));
+                      ++ optionals bssCfg.authentication.enableRecommendedPairwiseCiphers ["CCMP" "GCMP" "GCMP-256"]));
```

Comment:
This looks like a potential breaking change and deserves a release notes entry

Reply:
I don't think this is breaking, since CCMP (not CCMP-256) support is mandated by the wifi spec. So unless someone explicitly disabled CCMP on a device it'd just fall back to CCMP. But adding a release note won't hurt.